### ISSUE TITLE
"siac wallet transactions" show wrong block reward

### ISSUE THREAD

**Issue by wincss** [NebulousLabs/Sia#2384]
$ siac wallet transactions
    [height]                                                   [transaction id]    [net siacoins]   [net siafunds]
[... many transactions here ...]
      125614   000000000000000349466f24d07ad26cffc558bf1cf72f60ad17e677a64bbaff       94158.57 SC             0 SF
      125625   00000000000000160decd7adcea97d2c18ddd30d338670f2af430c1b3d0c5b22       94126.94 SC             0 SF
      125627   00000000000000314da8b3fae102e72df69246b4fda2e2f7f921802166dce3b1       94129.15 SC             0 SF
      125633   000000000000002f5f408319cf714940897bd5d7d07eb2a7679d7abd786e2de3       94120.87 SC             0 SF

These [net siacoins] of newly mined block reward is not equal to Miner Payouts in block explorer.

https://explore.sia.tech/block.html?height=125633

**Comment by DavidVorick** [comment]
Hmm. I'm not sure what would be causing this. Are you sure those transactions are miner payouts?

**Comment by wincss** [comment]
@DavidVorick yes, now it shows large minus siacoins like something is overflow...

also, different version shows different result:

In siad v1.2.2:
      134685   0000000000000020fbadc404045d7d6c88d28c56f9b5baf2f797c99f2a2d624a    -2635212.49 SC             0 SF
      134693   00000000000000467187ff2b558ce146a638b30ac2d53ca56a1355169350e97d    -2635240.51 SC             0 SF
      134695   000000000000004ed860c148c8b1119bc2ad547cc119d26bc375586f683979a7    -2635223.47 SC             0 SF
      134698   000000000000004b75bc6cc08c3225b7bb2c0f86f4a0193a7686b7b9faa67956    -2635219.30 SC             0 SF
      134712   000000000000002d506541b6ba58a03d270ae33b506d5e54401cf7ea8c49077b    -2635255.30 SC             0 SF
      134731   00000000000000617b732997af5806463e67a0ea5706de7aa82cf0e73317521d    -2635249.60 SC             0 SF

In siad v1.3.0:
      134685   0000000000000020fbadc404045d7d6c88d28c56f9b5baf2f797c99f2a2d624a    -5470481.32 SC             0 SF
      134693   00000000000000467187ff2b558ce146a638b30ac2d53ca56a1355169350e97d    -5470509.33 SC             0 SF
      134695   000000000000004ed860c148c8b1119bc2ad547cc119d26bc375586f683979a7    -5470492.29 SC             0 SF
      134698   000000000000004b75bc6cc08c3225b7bb2c0f86f4a0193a7686b7b9faa67956    -5470488.13 SC             0 SF
      134712   000000000000002d506541b6ba58a03d270ae33b506d5e54401cf7ea8c49077b    -5470524.12 SC             0 SF
      134731   00000000000000617b732997af5806463e67a0ea5706de7aa82cf0e73317521d    -5470518.43 SC             0 SF
